tests/consumers/test_consumer_builder.py

```diff
@@ -95,21 +101,23 @@ def test_consumer_builder_non_optional_attributes(con_build) -> None:  # type: i
     # Depending on the attribute, we can verify this
     # to different degrees
 
-    assert con_build.storage == get_writable_storage(test_storage_key)
+    storages = list(con_build.storages.values())
+    if len(storages) == 1:
```

Comment:
Can you add tests for the case where `len(storages) > 1`?

Reply:
Fixed